crypto/crmf/crmf_pbm.c

```diff
@@ -197,16 +200,21 @@ int OSSL_CRMF_pbm_new(OSSL_LIB_CTX *libctx, const char *propq,
      * function to be used.  All implementations MUST support HMAC-SHA1 [HMAC].
      * All implementations SHOULD support DES-MAC and Triple-DES-MAC [PKCS11].
      */
+    /*
+     * TODO It should be possible to fetch HMAC-SHA1 directly instead of
```

Comment:
No more TODO's in the code please - add an issue..

Reply:
I was worried that I wouldn't find the spot again, but found a way: https://github.com/openssl/openssl/issues/14543#issuecomment-797866558

So ok.